src/lib/helpers.ts

```diff
@@ -14,15 +14,31 @@ export const getSiteLink = ({
   subdomain: string
   noProtocol?: boolean
 }) => {
+  if (IS_VERCEL_PREVIEW || IS_DEV) return `/_site/${subdomain}`
+
   if (domain) {
     return `https://${domain}`
   }
   if (noProtocol) {
     return `${subdomain}.${OUR_DOMAIN}`
   }
+
   return `${IS_PROD ? "https" : "http"}://${subdomain}.${OUR_DOMAIN}`
 }
 
+export const getSlugUrl = (slug: string) => {
+  if (IS_VERCEL_PREVIEW || IS_DEV) {
```

Comment:
Why do we need it in the development environment? Do you also have subdomain issues during the development process?

Reply:
I will remove it

Reply:
I will remove it